src/workerd/server/tests/python/undefined-handler/server.mjs

```diff
@@ -4,8 +4,7 @@ export default {
   async test(ctrl, env) {
     await rejects(env.pythonWorker.fetch(new Request('http://example.com')), {
       name: 'Error',
-      message:
-        'Python entrypoint "undefined_handler.py" does not export a handler named "on_fetch"',
+      message: 'Handler does not export a fetch() function.',
```

Comment:
Don't we want to keep this custom message?

Reply:
Although hmm, I suppose I could check for `IS_WORKERD` and continue to add them all in that case.